src/server.js

```diff
@@ -11,7 +11,9 @@ const Security = require('./lib/security/security').getInstance();
 const KeyStorage = require('./lib/util/keyStorage');
 const TransactionBuilder = require('./lib/util/transactionBuilder').getInstance();
 const qrCodeGenerator = new QrCodeGenerator();
+const sleep = require('sleep');
 let io = require('socket.io');
+const Sentry = require('@sentry/node');
```

Comment:
Please conform to consistency, line 14 & 16 do not conform to the same capitalization.

Reply:
Resolved in 63ff4f3